### PR TITLE
docs(README): refactor for webpack v2

### PR DESCRIPTION
**Pull request by michael-ciniawsky**
[README](https://github.com/michael-ciniawsky/raw-loader/tree/readme)

@webpack/core-team @webpack/contributor-team

### CODE REVIEW

**Comment by SpaceK33z** [comment]
Hey, I updated your PR directly to make some changes. Hope you don't mind, I could make a PR to your fork next time if you want.

- Added a description to the loader. I think every loader needs to have a simple description. I can help you make them for the other loaders, since it can be quite unclear what some actually do.
- Used ES Modules (`import`) instead of `require`, since webpack v2 supports that out-of-the-box now and is the recommended way.
- Refactored "usage" part to make it clear that you should either configure it via your webpack config, or use it "manually". Do you maybe know a better word for "manually" in this case? I don't know how I should call the "manual" way of including `raw-loader!./file.txt` for each import.

**Comment by carlos-** [comment]
@SpaceK33z, I approved your changes too fast. Sorry. Could you change H2 tags to markdown?

**Comment by SpaceK33z** [comment]
@carlos-, that was not something I changed, @michael-ciniawsky is using the H2 tags on purpose because we can then center the text. It's fancy I think.

**Comment by carlos-** [comment]
@SpaceK33z yes, they add some stilling, but still I would disagree on adding custom styles. 
https://raw.githubusercontent.com/webpack/webpack/master/README.md has no center elements, just plain markdown.
@michael-ciniawsky?

**Comment by carlos-** [comment]
OK, I just saw the https://github.com/michael-ciniawsky/README 
If that is going to be the consistency, forget what I said. :grin:

**Comment by SpaceK33z** [comment]
Haha, yeah we (=core team) asked Michael to help with more consistent readme's across all loaders / plugins. There is also a PR open for the webpack/webpack repo which will bring the same styling.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@
   <h1>RAW Loader</h1>
 </div>
 
+A loader for webpack that lets you import files as a string.
```
Comment (michael-ciniawsky) [inline README.md]: That could be a subtitle , if they are a prerequisite for Loader README's, like in the webpack README, I will update it accordingly then :).

```
<h1>RAW Loader</h1>
<h3>A loader for webpack that lets you import files as a string.</h3>
```

File: README.md [hunk 2]
```diff
@@ -23,16 +25,11 @@ npm install --save-dev raw-loader
 
 <h2 align="center">Usage</h2>
 
-**require**
-```js
-const txt = require("raw!./file.txt")
-```
+Use the loader either via your webpack config or manually.
```
Comment (michael-ciniawsky) [inline README.md]: :+1: That was a missing piece to better distinguish between 'inline' (require) and webpack.config.js usage

File: README.md [hunk 3]
```diff
@@ -23,16 +25,11 @@ npm install --save-dev raw-loader
...
-```js
-const txt = require("raw!./file.txt")
-```
+Use the loader either via your webpack config or manually.
 
-**webpack.config.js**
-```js
-const txt = require('file.txt')
-```
+### Via webpack config (recommended)
```
Comment (michael-ciniawsky) [inline README.md]: ```<h2 align="center">Example</h2>``` ? 😛

**Comment by michael-ciniawsky** [comment]
@carlos- 😛 Yep, I understand your concerns too well, it took me a while to check what kind of HTML/Markdown 'bastardisation' works on github/npm, but that one does the job and is 'battle' tested so far e.g [postcss-load-config](https://www.npmjs.com/package/postcss-load-config) ```<h2 align="center">Chapter</h2>``` is ignored by npm and will just left align like markdown.

File: README.md (hunk 2 above)
Comment (SpaceK33z) [inline README.md]: 'inline', that is the word I was looking for!! That's more clear then 'manually'.

File: README.md (hunk 1 above)
Comment (SpaceK33z) [inline README.md]: Fixed.

**Comment by SpaceK33z** [comment]
@michael-ciniawsky I changed RAW -> Raw because it's not an acronym, and it might make people think it's a loader for [RAW files](https://en.wikipedia.org/wiki/Raw_image_format).

File: README.md (hunk 3 above)
Comment (michael-ciniawsky) [inline README.md]: if you want it to stay under the same (Usage) section normal markdown h3 `###` if it is on the 'top' level like Install, Usage, Options, [Example], Maintainer => `h2 align="center"></h2>`

**Comment by michael-ciniawsky** [comment]
yep, old habit of uppercasing all names  `text.length =< 4`, but you're absolutely right, this is a reserved word for RAW Camera files 😛

**Comment by SpaceK33z** [comment]
This one is ready. Linking to https://github.com/webpack/webpack/issues/3195 for visibility.

**Pull request merged**